pkg/build/build.go

```diff
@@ -252,6 +253,14 @@ func WithConfig(configFile string) Option {
 	}
 }
 
+// WithFailOnLint sets whether or not to fail on lint errors.
+func WithFailOnLint(fail bool) Option {
```

Comment:
I thought we had a way to make linter's warn by default.  This makes it a binary thing globally and we should be able to toggle each linter on as things are clean.

I'll take a stab at this.